MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include             LICENSE MANIFEST.in *.rst tox.ini docs-requirements.txt .coveragerc
+include             LICENSE MANIFEST.in *.rst tox.ini docs-requirements.txt .coveragerc conftest.py
```

Comment:
Shouldn't need this addition any more as the recursive-include on tests will catch it.